atest/robot/cli/dryrun/executed_builtin_keywords.robot

```diff
@@ -13,3 +13,18 @@ Set Library Search Order
     Should Be Equal    ${tc.kws[1].name}    Second.Parameters
     Should Be Equal    ${tc.kws[2].name}    First.Parameters
     Should Be Equal    ${tc.kws[4].name}    Dynamic.Parameters
+
+Set Tags
+    ${tc} =    Check Test Case    ${TESTNAME}
+    List Should Contain Value    ${tc.tags}    Tag0
+    List Should Contain Value    ${tc.tags}    Tag1
+    List Should Contain Value   ${tc.tags}    Tag2
+    List Should Contain Value   ${tc.tags}   Tag3
```

Comment:
We have a separate `Check Test Tags` keyword that could be used to replace all the above like
```
Check Test Tags    ${TESTNAME}    Tag0    Tag1    Tag2    Tag3
```

Reply:
Thanks for the help! Updated